clouddriver-docker/src/test/groovy/com/netflix/spinnaker/clouddriver/docker/registry/api/v2/client/DockerRegistryClientSpec.groovy

```diff
@@ -41,9 +41,9 @@ class DockerRegistryClientSpec extends Specification {
 
   void "DockerRegistryClient should validate that it is pointing at a v2 endpoint."() {
     when:
-      Boolean result = client.isV2()
+      client.isV2()
 
     then:
-      result
+      true
```

Comment:
I figured this would make it clear that it wasn't `boolean`, as I'm ignoring the (non existant) return value and making the test happy with an explicit `true`.